### ISSUE TITLE
Mathml is not read as math in live regions

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#7485]
Filed on behalf of @benetech
### Steps to reproduce:

1. go run [this example page](https://files.derekriemer.com/live_mathml.html). 
2. Wait ten seconds, then ten more seconds.

### Expected behavior:
NVDA should read the math in the live region.

### Actual behavior:
NVDA reads only the text of the live region, as if math doesn't exist.

### System configuration:
NVDA version: Doesn't matter

NVDA Installed or portable: installed or Portible.

Other information:
With any browser, and mathplayer.

Windows version:
Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:
mathplayer.

### Technical:

This codepath for live regions is not capable of handling this usecase. The code simply gets text with no formatting from the browser.

**Comment by ehollig** [comment]
@derekriemer, it appears the link that you provided no longer exists. Do you have another use case?

**Comment by derekriemer** [comment]
usecase exists now.

**Comment by bhavyashah** [comment]
I am unable to reproduce the reported issue with the provided test case. Every ten or so seconds, NVDA automatically reports something, and a quick snippet of Speech Viewer contents when the page is opened is as follows:
"lorem ibsum 
The power of x 1 2     is x to the powr of one over two.  
The fraction of x 2    is x devided by two.  
The power of x 1 2     is x to the powr of one over two. "
Isn't the above speech expected behaviour, or am I missing something? @derekriemer

**Comment by derekriemer** [comment]
No, this isn't real math. x2, not the fraction (Read that with mathplayer).

**Comment by pbblair** [comment]
Thought I'd link this up with a relevant conversation elsewhere:

[https://github.com/w3c/aria-practices/issues/696](https://github.com/w3c/aria-practices/issues/696)

Seems like there's interest in making this happen.

**Comment by Adriani90** [comment]
So I am understanding correctly that this one needs and external fix?

**Comment by pbblair** [comment]
External fix seems ideal, but would require more specific guidance for browsers on how to handle MathML within accessible names.

I wonder, since MathML is being read correctly elsewhere by NVDA/MathPlayer, is it feasible to ignore the browser's computed Accessible Name when math children are present?

**Comment by Adriani90** [comment]
cc: @NSoiffer

**Comment by seanbudd** [comment]
is this still an issue with 2026.1 betas which include mathcat?

**Comment by seanbudd** [comment]
We need the spec to change / the discussion to develop further before we can consider this
